edgexfoundry/device-rest-go
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rest-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) - bootstrap changes
- [ ] I have opened a PR for the related docs change (if not, why?) - docs for common config need to be added
  <link to docs PR>

## Testing Instructions

1. run edgex non-secure stack
2. stop the device rest container
3. build device rest - `make build`
4. run device rest using config provider
5. verify that private configuration is loaded in consul
6. ensure that writable in the private config is modifyable - change log level and see change in the terminal

## New Dependency Instructions (If applicable)
